users/models.py

```diff
@@ -5,11 +5,11 @@ class User(models.Model):
     last_name           = models.CharField(max_length=16)
     first_name          = models.CharField(max_length=32)
     email               = models.EmailField()
-    password            = models.CharField(max_length=256)
-    birthday            = models.DateField()
+    password            = models.CharField(max_length=256,default="a123456789")
```

Comment:
이건 왜 하시는거죠?  보안에 정말 취약한것 같은데요~!
```suggestion
    password            = models.CharField(max_length=256)
```

Reply:
get_or_create 방식의 이해부족으로 인하여
심각한 보안 이슈를 만들었습니다.

기존 방식으로 롤백하였고, 더욱 확실하게 이해한 뒤에 다시 구현하겠습니다.